doc/feature/validation.md

```diff
@@ -0,0 +1,80 @@
+Validation of the Local Potential Fields calculation
+====================================================
+
+Since LPF is a physically based computation we need a way to validate the
+results against the electromagnetic theory. We also should evaluate the impact
+of the cuttof distance on the errors between our computations and the ideal
+case i.e. infinite cutoff distance. Another step which is beyong pure algorithm
+validation would be to compute the errors between full compartments reports and
+soma only reports simulations.
+
+## Requirements
+
+* A simple setup( 1-5 point sources ) simulation should be compared to pure
+pencil/paper computation.
+* We need to create several graphs showing the errors decrease with the increase 
+of the cutoff distance.
+* We need to compute errors betweens full compartments reports and soma only 
+reports simulations.
+* We need to create a document to show the results to scientists.
+* Create tests.
+
+## Implementation
+
+To achieve all these validations, we need to have a volume exporter and a
+volume comparator.
+
+### Volume exporter
+
+The volume exporter must be very precise so one can easily map a voxel position
+into a real physical position. Any corner/border case errors must be avoided. 
+This is critical for example in the first requirement point above, one need to 
+compute by hand a field intensity value for a physical position, this physical 
+position should perfecty (in the range of floating point precision) match the 
+center of a voxel. Otherwise validation will be broken.
+
+Maybe it is possible to add some meta-data at the end of the file. Usefull 
+meta-data can be volume resolution(voxels/um), position of the circuit bounding
+box, origin, etc... The circuit bounding box can be very usefull to compare 
+volumes of different sizes especially knowing the fact that in our current 
+implementation different cuttof distance will generate different size volumes. 
+This point is open for discussion.
```

Comment:
This whole 'Volume exporter' is our current 'voxelize' app. Maybe you should make it explicit, that the changes will be done in the current tool, not adding a new one. All the improvements mentioned here should be included by default in my opinion, as it's always useful to have this kind of metadata, for example.